change/@microsoft-teams-js-adfbff51-25bd-4c3e-a634-5980e8eb69a0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Enabled `FrameContexts.tasks` for `startCall` API in `call.ts`",
```

Comment:
nit: it's `FrameContext.task` (no trailing 's'). Otherwise the change comment is beautiful!

Reply:
Hmm I see it as `FrameContexts.task`

Reply:
In the comment, it shows `FrameContexts.tasks`...?